### DISCUSSION TITLE
Install updates without version in separate folder

### DISCUSSION THREAD

**Discussion by ghost** [wireapp/wire-desktop#5075] [Ideas]
Current wire desktop application install every updates in new folder similar 3.9.29.28, 3.10.31.38 etc. Just that is not a issue at all for basic users However extremely annoying task for people who use policy based(rule based) software firewalls. ex: windows, comodo, symantec, eset etc

I expect no one will comment below for ignore implement this request by mentioning wire application based on electron or any other reasons.

**Comment by ffflorian** [comment]
Hi @ArchiveBase, to keep things simple, we are using Squirrel to install Wire on Windows but unfortunately, choosing your installation directory [is not supported by Squirrel](https://github.com/Squirrel/Squirrel.Windows/issues/696). Maybe it's possible to set up your firewall so that all exectuables from the Wire installation dir are allowed to access the internet?

> I expect no one will comment below for ignore implement this request by mentioning wire application based on electron or any other reasons.

If you have a better solution, let us know :wink:

**Comment by ghost** [comment]
Hi @ffflorian 
> choosing your installation directory is not supported by Squirrel

The developer aggressive however it seems looking new maintainer for that project

Ex: refer https://github.com/ramboxapp/community-edition Just it's installer wizard appear to choose whether current user or all users while wire only install under %appdata% then able configure silent installation without optional graphical wizard and update procedure may similar below)

![image](https://user-images.githubusercontent.com/46082456/70859864-7dfe1300-1f4c-11ea-8915-3199e99dba3c.png)

> Maybe it's possible to set up your firewall so that all exectuables from the Wire installation dir are allowed to access the internet?

It's not possible to set folder/dir on above mentioned most software firewalls. 😊

**Comment by ffflorian** [comment]
Thanks for the suggestions, I'll look into it!

**Comment by ghost** [comment]
@ffflorian Any update about this?
I see this's not that difficult as you guys late, probably no priority
